packages/docusaurus-plugin-content-blog/src/index.js

```diff
@@ -178,22 +170,9 @@ class DocusaurusPluginContentBlog {
         exact: true,
         modules: {
           content: source,
-          frontMatter: frontMatterPath,
           metadata: metadataPath,
-          prevItem:
-            prevItem != null
-              ? {
-                  metadata: prevItem.metadataPath,
-                  frontMatter: prevItem.frontMatterPath,
-                }
-              : null,
-          nextItem:
-            nextItem != null
-              ? {
-                  metadata: nextItem.metadataPath,
-                  frontMatter: nextItem.frontMatterPath,
-                }
-              : null,
+          prevItem: prevItem && prevItem.metadataPath,
```

Comment:
This will be `false` if non-existent. Need to be explicit about it.

Reply:
Hmm should we default it to null ?

prevItem is guaranteed to be null or a value
https://github.com/facebook/Docusaurus/blob/d80cf3d4c949fc48d25589f7a962d28ed82f79b5/packages/docusaurus-plugin-content-blog/src/index.js#L161-L162

Reply:
My bad, this is fine. I remembered wrongly.